app-frontend/src/app/components/mosaicMask/mosaicMask.html

```diff
@@ -0,0 +1,38 @@
+<div class="scene-list-pane">
+  <div class="sidebar-static">
+    <h5 class="sidebar-title">
+      Mask
+    </h5>
+    <div class="sidebar-actions">
+      <button class="btn btn-square" ng-click="$ctrl.closePanel()">
+        <i class="icon-cross"></i>
+      </button>
+    </div>
+  </div>
+  <div class="sidebar-scrollable">
+    <div class="instructions-container">
+      <button class="btn btn-primary"
+              ng-click="$ctrl.enableDrawToolbar()">
+        <i class="icon-pencil"></i>
+        Draw Mask
+      </button>
+      <p>
+        Select a drawing tool from the toolbar and click on the map to start drawing.
+        To edit a mask, select the edit icon. Click and drag points to move them, and double click
+        them to delete them.
+        To save your changes, click "Save"
+        To delete masks, either click the <i class="icon-trash"></i> icon on the map to select polygons, or delete them below.
+      </p>
+    </div>
+    <rf-mosaic-mask-item on-delete="$ctrl.onDeleteMask(mask)"
+                         mask="mask"
+                         ng-repeat="mask in $ctrl.maskList track by mask.createdAt">
```

Comment:
It'd be nice if there was a way to use `$ctrl.drawnPolygons` directly here; `$ctrl.maskList` is generated by a `$watchCollection`, and `ngRepeat` also uses a `$watchCollection` internally so this is triggering twice every time something changes. `ngRepeat` can iterate over an Object's (key, value) pairs, so maybe a plain-old Javascript object might be a better option than a `Map` for `drawnPolygons`. This is kind of a complex data flow so it may not work to use the same collection everywhere though.